fabric-key-binding-api-v1/src/main/java/net/fabricmc/fabric/api/client/keybinding/v1/KeyBindingHelper.java

```diff
@@ -46,6 +46,16 @@ public static KeyBinding registerKeyBinding(KeyBinding keyBinding) {
 		return KeyBindingRegistryImpl.registerKeyBinding(keyBinding);
 	}
 
+	/**
+	 * Returns if the given keybind has been registered
+	 *
+	 * @param keyBinding the keybinding
+	 * @return if the keybind has been registered
+	 */
+	public static boolean keybindingExists(KeyBinding keyBinding) {
```

Comment:
Why do you need this method? In general, you should just choose a specific key translation key that includes your mod id to avoid conflicts.

Reply:
it was an edge case in one of my mods,
my mod would dynamically add keybinds like Action 1, Action 2 and it was a good way to detect what action number to add next

if it needs to be removed thats fine